README.rst

```diff
@@ -2,14 +2,27 @@ VCenter Operator
 =============
 
 The VCenter Operator automatically configures and deploys cinder and nova-compute nodes corresponding to the discovered vcenters and clusters.
-It follows the convention over configuration principle to keep the configuration to a minimum. It relies heavily on a other configmaps being deployed by `openstack-helm <https://github.com/sapcc/openstack-helm>`_  and should be best deployed with it.
```

Comment:
Since you editing this line, could you please change: `s/configmaps/configmaps and secrets/`?